packages/cephalon/src/actions/tts.scope.ts

```
@@ -1,35 +1,38 @@
+import { createLogger, type Logger } from "@promethean/utils";
 
 import type { Bot } from "../bot.js";
-import { createLogger, type Logger } from "@promethean/utils/logger.js";
 
 export type TtsScope = {
-  logger: Logger;
-  policy: PolicyChecker;
-  speak: (input: TtsInput) => Promise<TtsOutput>;
+	logger: Logger;
+	policy: PolicyChecker;
+	speak: (input: TtsInput) => Promise<TtsOutput>;
```

ESLint

[error] Parameter should have an immutability of at least "ReadonlyDeep" (actual: "Mutable").

(functional/prefer-immutable-types)